packages/algob/src/lib/deployer.ts

```diff
@@ -85,13 +89,14 @@ export class AlgobDeployerImpl implements AlgobDeployer {
     return this.cpData.precedingCP[this.networkName].asa[name];
   }
 
-  async deployASC (name: string, source: string, account: Account): Promise<ASCInfo> {
+  async deployASC (name: string, scParams: Object, flags: ASCDeploymentFlags): Promise<ASCInfo> {
+    const result: ASCCache = await this.algoOp.ensuredCompiled(name, false);
+    const programb64 = result.compiled;
```

Comment:
Move console statements and programb64 to operator class.

Leave minimum amount of code here.

Make `this.algoOp.deployASC` accept `ASCCache` directly.

Reply:
Ok